packages/smart-contracts/src/lib/artifacts/ERC20SwapToPay/index.ts

```diff
@@ -20,6 +20,10 @@ export const erc20SwapToPayArtifact = new ContractArtifact<ERC20SwapToPay>(
           address: '0xb674e3d228e631594D8fd4BF947E1811288bf836',
           creationBlockNumber: 7363204,
         },
+        bsctest: {
```

Comment:
Same here.
You added twice the same address for two different versions. you should add the address only for the relevant version.

Reply:
thank you, done by keeping it only in the latest version.